edge-rtac/src/main/resources/domain/edge-rtac/features/util/initData.feature

```diff
@@ -0,0 +1,174 @@
+Feature: init data for edge-rtac
+
+  Background:
+    * url baseUrl
+    * callonce login { tenant: 'diku', name: 'diku_admin', password: 'admin' }
+    * def headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+
+
+  @PostModeOfIssuance
+  Scenario: create mode of issuance
+    * def modeOfIssuanceId = call random_uuid
+    * def instanceModeOfIssuanceRequest = read('samples/instance/issuance-mode-request-entity.json')
+    * instanceModeOfIssuanceRequest.name = instanceModeOfIssuanceRequest.name + ' ' + random_string()
```

Comment:
Why does a random string need to be appended to the name?

Reply:
**Serial** type issuance we already have in Diku tenant so creating random mode of issuance to consider it as a non-serial mode.

Reply:
> Serial type issuance we already have in Diku tenant so creating random mode of issuance to consider it as a non-serial mode.

Alternatively, use another of the existing modes of issuance that is appropriate for non-serial instances (ask the PO for which one) e.g. monograph.

Or, create a new mode of issuance with an appropriate name describing the intent (non-serial) rather than random.

Reply:
Yes I agree. Thankyou